core/modules/css/CssAccess.cc

```diff
@@ -511,29 +506,29 @@ CssAccess::getPartTableParams(std::string const& dbName,
         return params;
     }
 
-    // fill the structure
-    params.dirDb = paramMap["partitioning/dirDb"];
-    params.dirTable = paramMap["partitioning/dirTable"];
-    params.dirColName = paramMap["partitioning/dirColName"];
-    params.latColName = paramMap["partitioning/latColName"];
-    params.lonColName = paramMap["partitioning/lonColName"];
-    params.partitioned = paramMap.count("partitioning") > 0;
-    try {
-        auto iter = paramMap.find("partitioning/subChunks");
-        if (iter != paramMap.end()) {
-            params.subChunks = std::stoi(iter->second);
-        }
-        iter = paramMap.find("partitioning/overlap");
-        if (iter != paramMap.end()) {
-            params.overlap = std::stod(iter->second);
-        }
-    } catch (std::exception const& exc) {
-        LOGS(_log, LOG_LVL_ERROR, "one of the sub-keys is not numeric: "
-             << util::printable(paramMap));
-        throw KeyValueError(tableKey + "/partitioning", "one of the sub-keys is not numeric: " +
-                            std::string(exc.what()));
+    _fillPartTableParams(paramMap, params, tableKey);
+    return params;
+}
+
+ScanTableParams
+CssAccess::getScanTableParams(std::string const& dbName,
+                              std::string const& tableName) const {
+    LOGS(_log, LOG_LVL_DEBUG, "getScanTableParams(" << dbName << ", " << tableName << ")");
+    _checkVersion();
+
+    std::string const tableKey = _prefix + "/DBS/" + dbName + "/TABLES/" + tableName;
+
+    ScanTableParams params;
+
+    std::vector<std::string> subKeys{"sharedScan", "sharedScan/lockInMem", "sharedScan/scanSpeed"};
```

Comment:
"sharedScan" is not needed here, we are only using subkeys from this key. In case of "partitioning" key situation is different, we check the presence of that key to decide whether table is partitioned or not.